mysql-test/include/search_pattern_in_file.inc

```diff
@@ -52,23 +54,33 @@
 #
 # Created: 2011-11-11 mleich
 # Modified: 2014-04-26 hvadodar
+# Modified: 2017-11-10 rgolebiowski
```

Comment:
I don't like "changelog" comments like that: too easy not to update, do not add any value over git log/blame. But OK to keep since it's already here

Reply:
Noted.